packages/components/src/components/text-area/_text-area.scss

```diff
@@ -38,33 +43,33 @@
     transition: background-color $duration--fast-01 motion(standard, productive),
       outline $duration--fast-01 motion(standard, productive);
 
-    & ~ .#{$prefix}--form__helper-text {
-      margin-top: 0;
-      order: 2;
-      font-style: italic;
+    &:focus:not([readonly]),
```

Comment:
Can we add a reset style to negate focus outline for read-only variant?

Reply:
sure, added reset styles for read-only